docs/generated/packages/js/generators/library.json

```diff
@@ -95,24 +101,28 @@
         "type": "string",
         "enum": ["workspace", "project", "npm-scripts"],
         "default": "project",
-        "description": "Determines whether the project's executors should be configured in `workspace.json`, `project.json` or as npm scripts."
+        "description": "Determines whether the project's executors should be configured in `workspace.json`, `project.json` or as npm scripts.",
+        "x-priority": "internal"
       },
       "compiler": {
         "type": "string",
         "enum": ["tsc", "swc"],
         "default": "tsc",
-        "description": "The compiler used by the build and test targets"
+        "description": "The compiler used by the build and test targets",
+        "x-priority": "important"
       },
       "bundler": {
         "description": "The bundler to use.",
         "type": "string",
         "enum": ["none", "esbuild", "rollup", "vite", "webpack"],
-        "default": "none"
+        "default": "none",
+        "x-priority": "important"
       },
       "skipTypeCheck": {
         "type": "boolean",
         "description": "Whether to skip TypeScript type checking for SWC compiler.",
-        "default": false
+        "default": false,
+        "x-priority": "important"
```

Comment:
Is this really important? Will this be set by almost everyone when they use this executor?
I feel like that's not the case but you obv. know more about the actual usage than me.

Reply:
Not on the generator. Thanks for catching this